model/token-invalidated-event.js

```diff
@@ -0,0 +1,15 @@
+const {SingleUserEvent} = require("./single-user-event");
+const {TOKEN_CREATED} = require("../const/event-types");
+
+const TokenInvalidatedEvent = class extends SingleUserEvent{
+    constructor(user, token) {
+        super(user.ID, user.email, user.IDP);
+        this.event_type = TOKEN_CREATED;
+        this.token_uuid = token.uuid;
+        this.token_expiration = token.expiration;
```

Comment:
token expiration can be removed from this event type, since it has been invalidated in our database then it does not matter when it expires